tools/openblas_support.py

```diff
@@ -43,27 +43,26 @@ def download_openblas(target, arch, ilp64):
     fnsuffix = {None: "", "64_": "64_"}[ilp64]
     filename = ''
     if arch in ('aarch64', 'ppc64le', 's390x'):
-        filename = '{0}/openblas{1}-{2}-manylinux2014_{3}.tar.gz'.format(
-                        RACKSPACE, fnsuffix, OPENBLAS_LONG, arch)
+        filename = f'{RACKSPACE}/openblas{fnsuffix}-{OPENBLAS_LONG}-manylinux2014_{arch}'
         typ = 'tar.gz'
         typ = 'tar.gz'
     elif arch == 'darwin':
-        filename = '{0}/openblas{1}-{2}-macosx_10_9_x86_64-gf_1becaaa.tar.gz'.format(
-                        RACKSPACE, fnsuffix, OPENBLAS_LONG)
+        MAC_VERSION = 'macosx_10_9_x86_64-gf_1becaaa.tar.gz'
```

Comment:
To match the others below:
```suggestion
        suffix = 'macosx_10_9_x86_64-gf_1becaaa.tar.gz'
```